src/Core/ObjectRegistry.php

```diff
@@ -66,25 +66,27 @@
      * @param array $config Additional settings to use when loading the object.
      * @return mixed
      */
-    public function load($objectName, $config = [])
+    public function load($objectName, array $config = [])
```

Comment:
Read the `_checkDuplicate`, and try pass in a string.
**Boogeyman is in the corner** 
Don't just talk about the typehints.